index_search.go

```diff
@@ -38,6 +38,15 @@ func (i Index) Search(query string, request *SearchRequest) (*SearchResponse, er
 	if request.CropLength != 0 {
 		searchPostRequestParams["cropLength"] = request.CropLength
 	}
+	if request.CropMarker != "" {
+		searchPostRequestParams["cropMarker"] = request.CropMarker
+	}
+	if request.HighlightPreTag != "" {
```

Comment:
There is a way to automate this kind of thing, like serialize somehow?

Reply:
I would love it! I didn't find any other ways